python/python/glide/async_commands/core.py

```diff
@@ -1405,6 +1422,57 @@ async def type(self, key: str) -> str:
         """
         return cast(str, await self._execute_command(RequestType.Type, [key]))
 
+    async def geoadd(
+        self,
+        key: str,
+        members_coordinates: Mapping[str, Coordinate],
+        existing_options: Optional[ConditionalChange] = None,
+        changed: bool = False,
+    ) -> int:
+        """
+        Adds geospatial members with their positions to the specified sorted set stored at `key`.
+        If a member is already a part of the sorted set, its position is updated.
+
+        See https://redis.io/commands/geoadd for more details.
+
+        Args:
+            key (str): The key of the sorted set.
+            members_coordinates (Mapping[str, Coordinate]): A mapping of member names to their corresponding positions. See `Coordinate`.
+            The command will report an error when the user attempts to index coordinates outside the specified ranges.
+            existing_options (Optional[ConditionalChange]): Options for handling existing members.
+                - NX: Only add new elements.
+                - XX: Only update existing elements.
+            changed (bool): Modify the return value to return the number of changed elements, instead of the number of new elements added.
+
+        Returns:
+            int: The number of elements added to the sorted set.
+            If `changed` is set, returns the number of elements updated in the sorted set.
+
+        Examples:
+            >>> await client.geoadd("my_sorted_set", {"Palermo": Coordinate(13.361389, 38.115556), "Catania": Coordinate(15.087269, 37.502669)})
+                2  # Indicates that two elements have been added to the sorted set "my_sorted_set".
+            >>> await client.geoadd("my_sorted_set", {"Palermo": Coordinate(14.361389, 38.115556)}, existing_options=ConditionalChange.XX, changed=True)
```

Comment:
could be
```suggestion
            >>> await client.geoadd("my_sorted_set", {"Palermo": Coordinate(14.361389, 38.115556)}, ConditionalChange.XX, True)
```

Reply:
I am pretty sure we prefer adding arguments names in the examples, so it would be easier to undestand